frontend/src/bundles/common/pages/dashboard/dashboard.tsx

```diff
@@ -0,0 +1,220 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import { Button, Input } from '../../components/components';
+import { DoughnutChart } from '../../components/doughnut-chart/doughnut-chart';
```

Comment:
Fix import, should be from component.ts file
`import { Button, Input, DoughnutChart } from '../../components/components';`
My mistake, didn't add the Doughnut chart to the common components file

Reply:
Fixed in [https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/65/commits/36179e20e82c40c26e89ba2ef454742920f5dc18](url)